news/_posts/2021-01-07-next-official-release-spark-3.1.1.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Next official release: Spark 3.1.1" 
```

Comment:
How about 
```
Next official Spark release is 3.1.1 instead of 3.1.0
```

Reply:
I think the title is usually not a sentence. I thought the current title is concise enough but I don't mind changing it if others prefer.